docs/BigQueryMultiTable-batchsink.md

```diff
@@ -67,6 +67,10 @@ In this case, those fields in the BigQuery schema will be modified to become nul
                          
 Incompatible schema changes will result in pipeline failure.
 
+**Allow flexible schemas in Output**: When enabled, this sink will write out records with arbitrary schemas. 
+Records may not have a well defined schema depending on the source.
+When enabled, the BigQuery table schema will not be validated or updated.
```

Comment:
I see that update schema is hidden when this is set, but I don't see any changes in the logic around this. For example, if somebody deploys a pipeline through REST when both enabled. Is it handled somewhere?

Reply:
+1

Reply:
By default, the `allowFlexibleSchemas` option takes precedence. When `allowFlexibleSchemas` is enabled, the schema relaxation (updating the BQ schema) doesn't ever take place.

Should I return an error message if both options are enabled at the same time?

Reply:
I see that it doesn't get validated, but does BigQueryConstants.CONFIG_ALLOW_SCHEMA_RELAXATION still get set in the OutputFormat and used by the delegate? 

It would be good to try this scenario out (existing table with a different schema) to see what happens.

Reply:
should this line be removed now?

Reply:
Yes, I will update the docs.

Reply:
Updated.